packages/datadog-plugin-kafkajs/test/hash.spec.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
```

Comment:
Can this be tested using the plugin instead?

Reply:
I think you're saying that we're testing this functionality at a lower level, specifically the hash functionality itself, instead of at a higher level, being the Kafka/DMS code that calls the hashes.

We could go through the effort to refactor all these tests to move where the testing happens. But before we proceed, what would the value gained be?